lua/gitportal/cli.lua

```diff
@@ -31,11 +31,7 @@ function M.open_link_in_browser(link)
     return
   end
 
-  -- Construct the command to open the link in the default browser
-  local cmd = open_cmd .. " '" .. link .. "'"
-
-  -- Execute the command
-  os.execute(cmd)
+  vim.system({ open_cmd, link }):wait()
```

Comment:
vim.system automatically escapes the link which the old code didn't do, what if the link had a `'`

Reply:
I think I saw one other place in the codebase where the links are just concattenated with `/` and no regard if the link is valid or not (needs escaping)